src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceColumnManager.java

```diff
@@ -242,6 +243,23 @@ public void onSwitchToDoc(SwitchToDocEvent event)
          }
       });
 
+      userPrefs.allowSourceColumns().addValueChangeHandler(event ->
+      {
+         // PaneManager helps manage these commands so when modifying this code, update 
```

Comment:
It's not totally clear to me why we need this code in both places ... doesn't this handler run at startup? (if not, maybe make a function that's called both at startup and in the value change handler?)

Reply:
PaneManager just needs to help with commands that need to be adjusted based on the max column count. It tracks this because it's actually responsible for adding and removing the windows that contain the columns. I'll update the comment to be more specific. I was grouping all the commands into that category when I shouldn't have been.